flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -164,9 +164,14 @@
             ConfigOptions.key("taskmanager.state.local.root-dirs")
                     .noDefaultValue()
                     .withDescription(
-                            "The config parameter defining the root directories for storing file-based state for local "
-                                    + "recovery. Local recovery currently only covers keyed state backends. Currently, MemoryStateBackend does "
-                                    + "not support local recovery and ignore this option");
+                            Description.builder()
```

Comment:
Should this option be deprecated?

Reply:
My reasoning was that I wanted to give users still the option to configure a different local state directory. Especially if they want to configure multiple volumes, this is the only way now.